modules/likes/jetpack-likes-settings.php

```diff
@@ -237,14 +237,30 @@ function sharing_block() { ?>
 	 */
 	function is_post_likeable( $post_id = 0 ) {
 		$post = get_post( $post_id );
-		if ( !$post || is_wp_error( $post ) ) {
+		if ( ! $post || is_wp_error( $post ) ) {
 			return false;
 		}
 
 		$sitewide_likes_enabled = (bool) $this->is_enabled_sitewide();
 		$post_likes_switched    = get_post_meta( $post->ID, 'switch_like_status', true );
 
-		return $post_likes_switched || ( $sitewide_likes_enabled && $post_likes_switched !== '0' );
+		// on WPCOM, we need to look at post edit date so we don't break old posts
+		// if post edit date predates this code, stick with the former (buggy) behavior
+		// see: p7DVsv-64H-p2
+		$last_modified_time = strtotime( $post->post_modified_gmt );
+
+		// TODO set this to the current date/time immediately before committing
+		$behavior_was_changed_at = strtotime( "2019-02-20 00:00:00" );
```

Comment:
Remember to get server time, which is hopefully in UTC

Reply:
If I pull it from the Trac timestamp that should be correct I think -- at least a change from a few min ago is showing as 19:06 for me :)